pkg/controllers/jobset_controller.go

```diff
@@ -266,6 +267,7 @@ func (r *JobSetReconciler) calculateReplicatedJobStatuses(ctx context.Context, j
 				log.Error(nil, fmt.Sprintf("job %s missing ReplicatedJobName label", job.Name))
 			}
 		}
+		replicatedJobsReady[job.Labels[jobset.ReplicatedJobNameKey]]["active"] += job.Status.Active
```

Comment:
we need to track the number of active jobs not pods, so we should do the following outside this loop (like what we are doing for successful and failed below)

```
	for _, job := range jobs.active {
		replicatedJobsReady[job.Labels[jobset.ReplicatedJobNameKey]]["active"]++
	}
```

Reply:
I'm not sure that we need to do another loop to iterate over active jobs when we can do this already inside an existing one